suite-common/local-first-storage/src/storage/initEvoluKeysThunk.ts

```diff
@@ -1,7 +1,11 @@
+import { p256 } from '@noble/curves/nist';
```

Comment:
I am not sure which methods to use. I checked out `verifyAuthenticityProof` file where Martin said a similar thing is going on, but I couldn't make it work with the functions there, unfortunately.

So I imported these which I could make work.
Maybe, I'd export these fucntions from `@trezor-crypto`, but I am not sure.

I dont like the fact that these packages are in different version around here. Althought, I think this is a correct depencdey of the local-first-sotrage package

cc @peter-sanderson